# Generated by Django 3.1.1 on 2020-12-11 13:40

from django.db import migrations, models
import django.db.models.manager


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0032_validation_type_added'),
    ]

    operations = [
        migrations.CreateModel(
            name='Issue',
            fields=[
                ('name', models.CharField(max_length=50, primary_key=True, serialize=False)),
                ('self_url', models.CharField(max_length=255, null=True)),
                ('summary', models.CharField(max_length=255)),
                ('description', models.TextField(null=True)),
                ('status', models.CharField(max_length=255, null=True)),
                ('updated', models.DateTimeField(null=True)),
                ('product', models.CharField(max_length=255, null=True)),
                ('closed_reason', models.CharField(max_length=255, null=True)),
                ('root_cause', models.TextField(blank=True, null=True)),
                ('oses', models.JSONField()),
                ('exposure', models.PositiveSmallIntegerField(choices=[(4, 'Showstopper'), (3, 'High'), (2, 'Medium'), (1, 'Low'), (0, 'Undecided')], default=0)),
                ('components', models.JSONField()),
                ('platforms', models.JSONField()),
            ],
        ),
        migrations.DeleteModel(
            name='Issues',
        ),
        migrations.AlterModelManagers(
            name='validation',
            managers=[
                ('all_objects', django.db.models.manager.Manager()),
            ],
        ),
        migrations.AddField(
            model_name='result',
            name='issues',
            field=models.ManyToManyField(to='api.Issue'),
        ),
    ]
